components/eventing-controller/api/v1alpha1/subscription_conversion_unit_test.go

```diff
@@ -31,57 +30,57 @@ func Test_Conversion(t *testing.T) {
 		{
 			name: "Converting NATS Subscription with empty Status",
 			alpha1Sub: newDefaultSubscription(
-				testingv1.WithEmptyFilter(),
-				testingv1.WithEmptyConfig(),
-				testingv1.WithEmptyStatus(),
+				eventingtesting.WithV1alpha1EmptyFilter(),
```

Comment:
imho here v1Alpha1 should be part of the package as it was before. Is there a reason for this change?